core/src/test/java/org/apache/calcite/tools/PlannerTest.java

```diff
@@ -105,7 +107,8 @@ private void checkParseAndConvert(String query,
             + "FROM `emps`\n"
             + "WHERE `name` LIKE '%e%'",
 
-        "LogicalProject(empid=[$0], deptno=[$1], name=[$2], salary=[$3], commission=[$4])\n"
+        "LogicalProject(empid=[$0], deptno=[$1], name=[$2], salary=[$3], "
```

Comment:
no need to break long strings... check-style has an exception for lines that are just strings